modules/base/lightsource/scenegraphlightsource.cpp

```diff
@@ -41,18 +41,22 @@ namespace {
         openspace::properties::Property::Visibility::NoviceUser
     };
 
-    constexpr openspace::properties::Property::PropertyInfo NodeCameraStateInfo = {
+    constexpr openspace::properties::Property::PropertyInfo NodeInfo = {
         "Node",
         "Node",
         "The identifier of the scene graph node to follow.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    // This `LightSource` type represents a light source placed at the position of a
+    // scene graph node. That is, it will follow an existing object in the scene.
```

Comment:
```suggestion
    // scene graph node. That is, it will follow an existing object in the scene. Note
    // that the brightness of the light source does not depend on the distance between
    // the two scene graph nodes. Only the `Intensity` value has an impact on the
    // brightness.
```

?

Reply:
Good addition 👍

Reply:
Added your suggestion, and updated the description a bit to make it a bit clearer from that start that the _direction_  is what is determined from the other scene graph node. Let me know if it looks ok. 

Also, do you think we need to make the same clarification for the camera light source?

Reply:
I don't think we need to add it to the camera light source as people (a) don't generally change the FOV values and (b) wouldn't expect that the light changes based on that.   Using another scene graph node as the light source, since it lives in 3D space conjures more of the feeling of it being a "real" light source with falloff